src/Microsoft.Health.Fhir.Core/Data/OperationDefinition/export.json

```diff
@@ -0,0 +1,42 @@
+﻿{
+    "resourceType": "OperationDefinition",
+    "id": "export",
+    "url": "[base]/OperationDefinition/export",
+    "version": "1.0.0",
+    "name": "Export",
+    "title": "FHIR Bulk Data Export - System Level Export",
+    "status": "active",
+    "kind": "operation",
+    "description": "Export all data from a FHIR server. More details regarding bulk export can be found [here] [http://hl7.org/fhir/uv/bulkdata/export/index.html]. This operation is asynchronous as defined in the [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html)",
+    "code": "export",
+    "base": "/",
+    "system": true,
+    "type": false,
+    "instance": false,
+    "parameter": [
+        {
+            "name": "_outputFormat",
+            "use": "in",
+            "min": 0,
+            "max": "1",
+            "documentation": "The format for the requested bulk data files to be generated. Supported formats - `application/fhir+ndjson`.",
+            "type": "string"
+        },
+        {
+            "name": "_since",
+            "use": "in",
+            "min": 0,
+            "max": "1",
+            "documentation": "Resources updated after this period will be included in the response.",
+            "type": "instant"
+        },
+        {
+            "name": "_type",
+            "use": "in",
+            "min": 0,
+            "max": "1",
+            "documentation": "A string of comma-delimited FHIR resource types. Only resources of the specified resource types(s) will be included in the response.",
+            "type": "string"
```

Comment:
For the export docs, should we include our service specific parameters (container and the de-id params)?

Reply:
Added a separate operation definition for anonymized-export